packages/cli/src/test/integration/integration_test.ts

```diff
@@ -55,6 +55,26 @@ suite('integration tests', function() {
       await runCommand(binPath, ['test', '--module-resolution=node'], {cwd: dir});
     });
 
+    skipOnWindows('test the Polymer 3.x application template', async () => {
+      const dir =
+          await runGenerator(createApplicationGenerator('polymer-3.x'))
+              .withPrompts({name: 'my-app'})  // Mock the prompt answers
+              .toPromise();
+      // TODO(#118): Use `polymer install` once it supports installing npm
+      // packages.
+      await exec('npm install', {cwd: dir});
```

Comment:
(This PR might need to wait on #136 because of this fix: https://github.com/Polymer/tools/pull/136/files#diff-10eed550dca082f37ae2a959b6f87823R29.)

Reply:
Nope, #136 safely rewrites the `exec` used here. :)